docs/releasenotes/3.3.0.rst

```diff
@@ -84,6 +84,37 @@ Our internal reports structure was refactored to make it easier to maintain and
 each report is stored in a separate file in Robocop project. From the user's perspective, reports should work
 the same as before.
 
+Rule severity thresholds printed value (#791)
+-----------------------------------------------
+
+Rule severity threshold allows to change rule severity depending on the threshold value and configured severities.
+In fact, it was confusing to see the reports for the same rule with different severities but the same limit value.
+We changed this behaviour to log the actual value that met the severity threshold.
+
+As for example, you can configure ``too-many-calls-in-test-case`` to report warning on 10 keywords and error on 30
+keywords. With such configuration you will get following reports::
+
+    test.robot:33:1 [W] 0505 Test case 'Test case 1' has too many keywords inside (24/10) (too-many-calls-in-test-case)
+    test.robot:62:1 [E] 0505 Test case 'Test case 2' has too many keywords inside (30/30) (too-many-calls-in-test-case)
+
+Not configured severity threshold should be ignored (#791)
```

Comment:
I just realized that this change is backward-incompatible. Am I right?

This is not a big change that affects how Robocop works in general, but it affects the configuration of severity for the rules. I am thinking if that forces us to make a major release 4.0.0, or can we just add it to release notes as backward-incompatible and go with 3.3.0?

@bhirsz What do you think?

Reply:
It's not entirely backward incompatible - depends on how we define it :) In the past we released major versions for changes that **breaks** the code, make configuration invalid or big changes to the tool etc. This change requires users to update the severity thresholds if they want them to catch all issues (in case they omitted default severity in severity threshold config). But it doesn't break anything, we simply changes how it behaves. I will go with 3.3.0 but add a note that this change requires updating current configuration

Reply:
Updated in #885